crates/rolldown_common/src/types/import_record.rs

```diff
@@ -86,8 +91,13 @@ pub struct ImportRecord {
   pub module_request: Rstr,
   pub resolved_module: ModuleIdx,
   pub kind: ImportKind,
+  /// We will turn `import { foo } from './cjs.js'; console.log(foo);` to `var import_foo = require_cjs(); console.log(importcjs.foo)`;
+  /// `namespace_ref` represent the potential `import_foo` in above example. It's useless if we imported n esm module.
   pub namespace_ref: SymbolRef,
+  /// If it is `import * as ns from '...'` or `export * as ns from '...'`
   pub contains_import_star: bool,
+  /// If it is `import def from '...'`, `import { default as def }`, `export { default as def }` or `export { default } from '...'`
```

Comment:
what about rename these two field to `contains_star` and `contains_default` ?

Reply:
There are indeed treat as `import_xxx`

https://github.com/rolldown/rolldown/blob/f790e8ce7eb1fde427680e0f640f40912994fbb0/crates/rolldown/src/ast_scanner/mod.rs#L255-L289.

`contains_star`  might be confused with `export * from '..'`